src/Chapter1.hs

```diff
@@ -503,7 +503,7 @@ branches because it is an expression and it must always return some value.
   satisfying the check will be returned and, therefore, evaluated.
 -}
 closestToZero :: Int -> Int -> Int
-closestToZero x y = error "closestToZero: not implemented!"
+closestToZero x y = if abs x == min (abs x) (abs y) then x else y
```

Comment:
Interesting solution 🧠 

However, this could be slightly simplified, and the amount of function calls reduced.
First of all, I notice that you use `abs x` twice. That means it makes sense to have it in a separate variable (either with `where` clause, or `let-in` construction.

But actually, you don't need to compare the minimum, it is enough to compare absolute values of both number and tell the result straightaway:

```haskell
if abs x < abs y then x else y
```

Reply:
nice reduced!!